packages/frontend-core/src/utils/relatedColumns.js

```diff
@@ -0,0 +1,111 @@
+import { FieldType, RelationshipType } from "@budibase/types"
+import { Helpers } from "@budibase/bbui"
+
+const columnTypeManyTypeOverrides = {
```

Comment:
We don't have unit test in frontend-core. It might be a good opportunity to add them, but I will add them separately and it needs few changes just for the vitest setup